core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -2314,7 +2315,8 @@ public UUID javaUuid() {
 
     @Override
     public boolean isLinked() {
-        return false; //todo
+        // Java and linked players' uuid version is 4, while bedrock is 0
+        return this.javaUuid().version() == 4;
```

Comment:
With local linking this isn't necessarily true.. might be easier to check if the uuid *isn't* version 0?